src/components/memorygame/gameHelpers.jsx

```diff
@@ -0,0 +1,46 @@
+// manipulates isFlipped and isMatched property of card a state
+// property example : "isMatched"
+// cardStates is an array of objects(states)
+// see src/memorygame/MemoryGameBoard/index.jsx to for cardStates example and use of this function
+// targetCard is an element of cardStates array.
+// changeCardStatePropertyToOpposite is called when selectedCards state change.
+export const changeCardStatePropertyToOpposite = (
+  stateProperty,
+  targetCard,
+  cardStates,
+  setCardStates
+) => {
+  const copyOfCardStates = [...cardStates];
+  const targetIndex = copyOfCardStates.findIndex(
+    (cardState) => cardState.cardKey === targetCard.cardKey
+  );
+  const targetCardState = copyOfCardStates[targetIndex];
+  // isMatched is false as default. When it is true success symbol appears in card
+  if (stateProperty === "isMatched") {
+    targetCardState.isMatched = !targetCardState.isMatched;
+    setCardStates(() => [...copyOfCardStates]);
+    // isFlipped is false as default. When it is true backside of card become visible
```

Comment:
don't put comments above else like this. it's confusing because it's in the if condition above it. put it below the else

Reply:
you can just delete this comment I think. no need to describe isflipped here, islfipped is probably the most intuitive variable in the code

Reply:
comment deleted PTAL